### ISSUE TITLE
Errors found if a `find` with no results is piped in

### ISSUE THREAD

**Issue by pecuchet** [JamieMason/ImageOptim-CLI#119]
Hello,
I'm using ImageOptim in a `bash` script. It gives me an error when used in combination with a `find` command that doesn't return anything. For instance:
`$ find "/path/to/folder/without/gifs" -name '*.gif' | imageoptim`
 The error message is the following:

```
sort: open failed: /var/folders/nd/klg579_d44g9xkmlv9dsmsc80000gn/T//imageoptim-cli/index.txt: No such file or directory
Running ImageOptim...
Runtime error (func=(main), adr=9): Divide by zero
(standard_in) 2: parse error
TOTAL was: 0kb now: 0kb saving: 0kb (%)
```

I tried to use `imageoptim` with `find -exec` like in `find "path" -name '*.gif' -exec imageoptim {} +` but ImageOptim doesn't support this.

Any idea on what I could do to prevent the error message?

**Comment by JamieMason** [comment]
Thanks @pecuchet, I think some hardening is needed around the cases where no images are passed — it's a corner case I've overlooked.

**Comment by JamieMason** [comment]
You could `imageoptim --directory "/path/to/folder/without/gifs"`, but that would process everything, not just the GIFs.

**Comment by jamesstout** [comment]
Quick PR to address this: #147

**Comment by JamieMason** [comment]
Released in [1.15.1](https://github.com/JamieMason/ImageOptim-CLI/blob/master/CHANGELOG.md#1151-2017-06-29) 🎉 Thanks to @jamesstout
